packages/backpack-core/config/webpack.config.js

```diff
@@ -115,7 +115,14 @@ module.exports = (options) => {
       new webpack.BannerPlugin({
         raw: true,
         entryOnly: false,
-        banner: `require('${require.resolve('source-map-support/register')}')`
+        banner: `require('${
+            // Is source-map-support installed as project dependency, or linked?
+            require.resolve('source-map-support').match(process.cwd())
```

Comment:
@sompylasar Can you clarify again?  I considered `indexOf`, but found `String.prototype.match` to read better.

Thanks!